benchmarks/lasso/solvers/glmnet.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    import rpy2.robjects.packages as rpackages
+    from rpy2 import robjects
+    from rpy2.robjects import numpy2ri
+
+
+class Solver(BaseSolver):
+    name = "glmnet"
+
+    install_cmd = 'bash'
+    install_script = 'install_glmnet.sh'
+    requirements_import = ['rpy2']
+
+    def set_objective(self, X, y, lmbd):
+        self.X, self.y, self.lmbd = X, y, lmbd
+        self.lmbd_max = np.max(np.abs(X.T @ y))
+
+    def run(self, tol):
+        numpy2ri.activate()
+        rpackages.importr('glmnet')
+        glmnet = robjects.r['glmnet']
```

Comment:
I would put this in `set_objective` as this is doing imports that can take time while not related to the benchmark in itself.